genny/ci/templates/-dot-circleci/config.yml.tmpl

```diff
@@ -0,0 +1,62 @@
+version: 2
+jobs: 
+  build:
+    docker:
+      - image: circleci/golang:1.12
+      {{ if eq .opts.DBType "postgres" -}}
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: {{.opts.App.Name.File}}_test
+      {{- end }}
+
+    environment:
+      TEST_RESULTS: /tmp/test-results
+
+    steps:
+      - checkout
+      - run: mkdir -p $TEST_RESULTS
+
+      - restore_cache:
+          keys:
+            - v1-pkg-cache
+
+      - run: go get github.com/jstemmer/go-junit-report
+
+      {{ if eq .opts.DBType "postgres" -}}
+      - run: go get github.com/lib/pq
+      - run: go get github.com/mattes/migrate
+      - run:
+          name: Waiting for Postgres to be ready
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for Postgres && exit 1
+      {{- end }}
+
+      - run:
+          name: Install Buffalo
+          command: |
+            go get github.com/gobuffalo/buffalo/buffalo
+            go get $(go list ./... | grep -v /vendor/)
```

Comment:
Buffalo assumes go modules since December last year. We should pull dependencies with `go mod download` or these would be downloaded at compile time.

Reply:
That's an easy fix. Will push an update.

Reply:
BTW - The TravisCI template needs updating too then as it still seems pre-modules.

Reply:
Thanks! Sorry to be holding this one. I'm very excited about getting it merged. Appreciate you putting time on this.

Reply:
Okay - simplified the template and updated the Travis and GitLab templates for modules.